src/olympia/users/urls.py

```diff
@@ -3,13 +3,18 @@
 from django.views.generic.base import RedirectView
 
 from session_csrf import anonymous_csrf
+from waffle.decorators import waffle_switch
 
 from . import forms, views
 
 
 USER_ID = r"""(?P<user_id>[^/<>"']+)"""
 
 
+def migration_on(fn):
+    return waffle_switch('!fxa-migrated')(fn)
```

Comment:
Yeah, I was disappointed when I saw the signature for `waffle_switch` and it didn't accept an `active` kwarg. Then I saw that and didn't quite know how to feel. At least my problem was solved I guess.